src/applications/mhv/secure-messaging/tests/e2e/fixtures/generalResponses/user.json

```diff
@@ -130,10 +130,10 @@
   "meta": {
     "errors": [
       {
-        "externalService": "EMIS",
+        "externalService": "Vet360",
         "startTime": "2023-08-22T16:15:17Z",
         "endTime": null,
-        "description": "EMISRedis::VeteranStatus::RecordNotFound, NOT_FOUND",
+        "description": "fake error",
```

Comment:
Can you provide context for this description?

Reply:
i believe the emis error was a legitimate error that was seen when this user response was generated. I wanted to make it clear that the Vet360 error was not a real one seen in the wild, so that's why I opted for "fake error". I would have just removed the error block, but from my testing it appeared that a test needed the error block.

Reply:
All good, the purpose seems somewhat vague if a developer comes across it in the future. Perhaps there's something more precise like "Expected Error Description Placeholder" or "Mock Error Description for Testing"?

Reply:
Sure, I can do that 👍